examples/kubernetes/static_provisioning/README.md

```diff
@@ -18,50 +33,64 @@ The CSI driver only supports static IAM credentials, which can be provided in tw
 1. **Driver-level authentication** (default): Credentials are configured at the driver level and used for all PVs
    - Set using environment variables in the driver deployment
    - Configured via Helm values or directly in the deployment YAML
-   
+
 2. **Secret-level authentication**: Credentials are provided per-volume using Kubernetes Secrets
    - See `secret_authentication.yaml` for an example
    - Use `authenticationSource: secret` in volumeAttributes
    - Reference a secret with `nodePublishSecretRef`
 
 ## AWS Endpoint URL Configuration
-For security and consistency reasons, if `--endpoint-url` is specified in the `mountOptions` of a PersistentVolume, it will be **ignored** by the driver. This is enforced in both systemd and pod mounters to prevent potential security risks like endpoint redirection attacks.
+
+For security and consistency reasons, if `--endpoint-url` is specified in the `mountOptions` of a PersistentVolume,
+it will be **ignored** by the driver. This is enforced in both systemd and pod mounters to prevent potential security
+risks like endpoint redirection attacks.
 
 To configure a custom endpoint URL for S3 requests, you must set it at the driver level using one of the following methods:
 
 ### Using Helm
+
 ```yaml
 # values.yaml for Helm chart
 node:
   s3EndpointUrl: "https://s3.example.com:8000"
 ```
 
 ## Configure
+
 ### Edit [Persistent Volume](https://github.com/scality/mountpoint-s3-csi-driver/blob/main/examples/kubernetes/static_provisioning/static_provisioning.yaml)
-> Note: This example assumes your S3 bucket has already been created. If you need to create a bucket, follow the [S3 documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html).
+
+> Note: This example assumes your S3 bucket has already been created. If you need to create a bucket, follow the
+> [S3 documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html).
+
```

Comment:
If you have [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admon#configuration) configured into your `mkdocs.yml` file. This syntax makes it look pretty nice:

```suggestion
!!! note

    This example assumes your S3 bucket has already been created. If you need to create a bucket, follow the
    [S3 documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html).

```

If linter complains, checkout if there's anything I have [here](https://github.com/scality/devdocs/blob/main/.trunk/configs/.markdownlint.yaml) that you don't.

Reply:
Nice, thank you, I adopted it in [the latest commit](https://github.com/scality/mountpoint-s3-csi-driver/pull/53/commits/c18ed9dfe16212ceee5171a49b45673756a308cd)
I didn't need to change anything on the rules side